app/views/content/non-uk-teachers/visas-for-non-uk-trainees.md

```diff
@@ -36,208 +36,122 @@ keywords:
   
 ---
 
-If you’re applying for teacher training as a non-UK citizen, you will need a visa or immigration status allowing you to study or work in England. 
+If you’re applying for teacher training as a non-UK citizen, you will need a [visa or immigration status allowing you to study or work in England](#check-your-status). 
```

Comment:
I think this text may get lost. 

For me it seems more logical to have the 'check your status' content as the first H2. Or to move this content under the 'How to apply for your visa' heading?

Reply:
Makes sense. I don't know if it's style to have some content under the yellow h1 before using an h2? 
On the order, the vast majority of people will fall into the Student visa bucket so I want to prioritise that content.